mantidproject/mantid
# ScriptRepo now works with the new internetHelper

Ticket http://trac.mantidproject.org/mantid/ticket/11069
# To Test
1. Use the script repository
2. Download plenty of files
3. Upload a file
   1. By default it is setup to load from the live repo, but upload to a seperate one, so you should get success, but if you refresh the dialog you will get the icon reset as if it was not uploaded.
4. Delete a file
   1. By default it is setup to load from the live repo, but upload to a seperate one, so you should get success, but if you refresh the dialog you will get the icon reset as if it was not uploaded.
